mmv1/third_party/terraform/services/vertexai/resource_vertex_ai_feature_online_store_featureview_test.go

```diff
@@ -0,0 +1,200 @@
+			{
+				ResourceName:            "google_vertex_ai_feature_online_store_featureview.featureview",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"name", "etag", "feature_online_store", "labels", "terraform_labels"},
+			},
+			{
+				Config: testAccVertexAIFeatureOnlineStoreFeatureview_vertexAiFeatureonlinestoreFeatureview_update(context),
+			},
+			{
+				ResourceName:            "google_vertex_ai_feature_online_store_featureview.featureview",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"name", "feature_online_store", "labels", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccVertexAIFeatureOnlineStoreFeatureview_vertexAiFeatureonlinestoreFeatureview_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_vertex_ai_feature_online_store" "featureonlinestore" {
+  name     = "tf_test_featureonlinestore12"
+  labels = {
+    foo = "bar"
+  }
+  region   = "us-central1"
+  bigtable {
+   auto_scaling {
+   min_node_count = 1
+   max_node_count = 2
+   cpu_utilization_target = 80
+ }
+}
+}
+
+resource "google_bigquery_dataset" "tf-test-dataset" {
+
+  dataset_id                  = "tf_test_dataset1_featureview12"
+  friendly_name               = "test"
+  description                 = "This is a test description"
+  location                    = "US"
+}
+
+resource "google_bigquery_table" "tf-test-table" {
+  deletion_protection = false
+
+  dataset_id = google_bigquery_dataset.tf-test-dataset.dataset_id
+  table_id   = "tf_test_bq_table"
+  schema = <<EOF
+  [
+  {
+    "name": "entity_id",
+    "mode": "NULLABLE",
+    "type": "STRING",
+    "description": "Test default entity_id"
+  },
+    {
+    "name": "test_entity_column",
+    "mode": "NULLABLE",
+    "type": "STRING",
+    "description": "test secondary entity column"
+  },
+  {
+    "name": "feature_timestamp",
+    "mode": "NULLABLE",
+    "type": "TIMESTAMP",
+    "description": "Default timestamp value"
+  }
+]
+EOF
+}
+
+resource "google_vertex_ai_feature_online_store_featureview" "featureview" {
+   name = "tf_test_tf_test_fv1%{random_suffix}"
+   region   = "us-central1"
+   labels = {
+    foo = "bar"
+  }
+    feature_online_store = google_vertex_ai_feature_online_store.featureonlinestore.name
+    sync_config {
+      cron = "0 0 * * *"
+    }
+    big_query_source {
+        uri = "bq://${google_bigquery_table.tf-test-table.project}.${google_bigquery_table.tf-test-table.dataset_id}.${google_bigquery_table.tf-test-table.table_id}"
+        entity_id_columns = ["test_entity_column"]
+
+    }
+}
+
+data "google_project" "project" {
+  provider = google
+}
+`, context)
+}
+
+func testAccVertexAIFeatureOnlineStoreFeatureview_vertexAiFeatureonlinestoreFeatureview_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_vertex_ai_feature_online_store" "featureonlinestore" {
```

Comment:
Spacing from here down looks broken

Reply:
Ran `go fmt` again.

Reply:
Looks like they continue to be oddly spaced. I don't think go fmt will work on this file as this is a terraform config represented as a string, so the formatter doesn't know how to handle it

Reply:
Ah ok. I've ran the terraform formatter separately on the terraform portions and added it back. pls check now.